README.md

```diff
@@ -7,12 +7,65 @@ MyHebrewDates is a web app for sharing Hebrew dates like birthdays, anniversarie
 
 License: MIT
 
+## Contributors
+<div align="center">
+<a href="https://github.com/abe-101/myHebrewDates/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=abe-101/myHebrewDates" />
+</a>
+</div>
+
 ## Settings
```

Comment:
remove this section

Reply:
as u have mentioned for the credit of contributers